src/utils/misc/index.ts

```diff
@@ -0,0 +1,22 @@
+export const formDataToJSON = <T>(formData: FormData) => {
+  const json: {
+    [key: string]: string | File | boolean | string[] | File[] | number;
+  } = {};
+  for (const [key, value] of formData.entries()) {
+    if (formData.getAll(key).length > 1) {
```

Comment:
There are some cases where you will have to send a single element on the array if the API server expects to receive an array

Reply:
Then, see my last comment.

Reply:
Now it should work with any `contacts_list` length.
With 0 `redirect_uris` it throws 400 bad request asking for at least 1 `redirect_uri`.
I think that for the moment it is ok, we will implement a proper error/notification system for failures.